woocommerce/woocommerce-gutenberg-products-block
# Add stylelint to find and fix css code style issues

In this PR, I've added `stylelint` and [the gutenberg `stylelint` config,](https://github.com/WordPress/gutenberg/blob/master/.stylelintrc.json) then fixed all the errors this generated.

**To test**

1. Run `npm install` to add the stylelint dependencies
2. Run `npm run lint:css` to check for errors.
3. Expect: There are no errors
4. You can also run `npm run lint` to run both the CSS and JS linters– the JS linter will return warnings, but there should be no errors.
